src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/template.json

```diff
@@ -1026,6 +1065,33 @@
           "exclude": [
             "MyExtensionsApp/appsettings.*"
           ]
+        },
+        {
+          "condition": "(useAzureDevOps)",
+          "rename": {
+            "build": "azure"
+          },
+          "exclude": [
+            "github/**/*"
+          ]
+        },
+        {
+          "condition": "(useGithubActions)",
+          "rename": {
+            "github": ".github"
+          },
+          "exclude": [
+            "azure-pipelines.yml",
+            "build/**/*"
```

Comment:
also let's change this directory back to Azure. We can do the rename like we are for github. This makes it easier to manage long term if we have other options in the future like Nuke Build that would also want a build directory.